packages/common/src/ui/components/inputs/CurrencyInput/CurrencyInput.tsx

```diff
@@ -7,7 +7,7 @@ import { useCurrency } from '@common/intl';
 
 interface CurrencyInputProps extends RCInputProps {
   onChangeNumber: (value: number) => void;
-  maxWidth?: number;
+  maxWidth?: number | string;
```

Comment:
Included string so other units can be passed down (%, ems, etc) as per standard css.